modules/cassandra/src/test/java/org/testcontainers/containers/CassandraContainerTest.java

```diff
@@ -107,7 +110,31 @@ public void testCassandraQueryWaitStrategy() {
     public void testCassandraGetCluster() {
         try (CassandraContainer<?> cassandraContainer = new CassandraContainer<>()) {
             cassandraContainer.start();
-            ResultSet resultSet = performQuery(cassandraContainer.getCluster(), "SELECT release_version FROM system.local");
+            ResultSet resultSet = performQuery(cassandraContainer.getCluster(), BAISC_QUERY);
+            assertTrue("Query was not applied", resultSet.wasApplied());
+            assertNotNull("Result set has no release_version", resultSet.one().getString(0));
+        }
+    }
+
+    @Test(expected = IllegalStateException.class)
+    public void testMissingLocalDatacenter() {
+        try (CassandraContainer<?> cassandraContainer = new CassandraContainer<>(CASSANDRA_IMAGE)) {
+            cassandraContainer.start();
+            // Trying to build a CqlSession will fail if a Contact Point is specified,
+            // but Local Datacenter is omitted.
+            CqlSession.builder().addContactPoint(cassandraContainer.getContactPoint()).build();
```

Comment:
Maybe instead of `@Test(expected = IllegalStateException.class)`, we could do a more detailed verification by catching an exception and validating that the exception message contains `contact-points` string? Or use AssertJ `assertThatThrownBy`

Reply:
I've changed the test to verify part of the exception message here: https://github.com/testcontainers/testcontainers-java/pull/2830/files#diff-f060cde9ce304b8f1959fc12ce03a851R131-R132